app/assets/javascripts/Components/dashboard.jsx

```diff
@@ -4,6 +4,85 @@ import { render } from 'react-dom';
 import { Bar } from 'react-chartjs-2';
 
 
+class Assignment extends React.Component {
```

Comment:
This component name is too broad---I suggest checking with the group and trying to come up with a common naming style for your new components.